sdk/eidolon_ai_sdk/cpu/llm_unit.py

```diff
@@ -67,6 +67,7 @@ class LLMCallFunction(BaseModel):
 
 class LLMUnitSpec(BaseModel):
     model: str = Field(description="The model to use for the LLM.")
+    supported_models: Optional[List[LLMModel]] = Field(description="The list of supported models or leave empty for defaults.")
```

Comment:
This needs a default or it will blow up on instantiation. We could do without the Optional here as well. List[llmmodel] | Literal["default"] = "default" is probably the most explicit